source/pug/views/style-guide.pug

```diff
@@ -76,8 +76,8 @@ block content
         h4 .join-us.full-page
         .join-us.full-page
 
-        h4 .pulse-project-list(for="QUERY")
-        .pulse-project-list(for=`mozfest`)
+        h5 .pulse-project-list(for="SEARCH_QUERY", size="MAX_TO_DISPLAY", rev="BOOLEAN_TO_INDICATE_ORDERING")
```

Comment:
this should stay as `h4`?

Reply:
Doesn't really matter...just the style guide. It was kinda large as a H4.